### DISCUSSION TITLE
Write the "Using custom encypted passwords" guide

### DISCUSSION THREAD

**Discussion by guidograzioli** [ansible-middleware/amq#232] [General]
##### SUMMARY

The guide [Using custom encypted passwords](http://ansiblemiddleware.com/amq/main/masked_passwords.html) needs a tester and technical writer. Experiment and analyze the molecule test [here](https://github.com/ansible-middleware/amq/tree/main/molecule/mask_passwords), and using it write down the guide in markdown [here]( https://github.com/ansible-middleware/amq/blob/main/docs/masked_passwords.md).

##### ISSUE TYPE
 - Documentation

**Comment by ericmchen15** [comment]
ericmchen15

**Comment by RanabirChakraborty** [comment]
@ericmchen15 how's the work going? do you need any help on this?
